libraries/constants_utils.py

```diff
@@ -0,0 +1,157 @@
+def generate_library_docs_url(boost_url_slug, library_slug):
```

Comment:
why move all this? and where's the copyright notice

Reply:
After I'd moved the constants in `views.py` to `constants.py` and the functions from the `LibraryList` class that would only be used in the dispatch method, but not any other `LibraryList` related method, seem like more appropriate places for them, I started getting a circular import between `constants.py` and `utils.py`. 

This was because of those methods which were in `utils.py`, with `utils.py` trying to import the constants from `constants.py` and those constants then needing an import of those methods to generate the constant dict.

To break the loop, and with all those functions being specific to constant generation it made sense to move them to `constants_utils.py`.

What copyright notice are you expecting? I've looked at other PRs and other files and only see some on three standalone shell scripts and some of the CSS files.